documentation/src/test/java/example/testinterface/TestLifecycleLogger.java

```diff
@@ -25,7 +25,7 @@
 @TestInstance(Lifecycle.PER_CLASS)
 interface TestLifecycleLogger {
 
-	static final Logger logger = Logger.getLogger(TestLifecycleLogger.class.getName());
+	Logger logger = Logger.getLogger(TestLifecycleLogger.class.getName());
```

Comment:
I wouldn't say these keywords were superfluous. There's no reason to make a logger per instance and I don't see why anyone would want to reassign the logger.

Reply:
Of course, you are right :) And also I have. Fields in an interface are by default public, static and final.